_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Business Analyst (BA)"
```

Comment:
maybe the title could be more specific like "Life of a Business Analyst", "Tea Chat with a Business Analyst" because I didn't know what the article was going to be about until I started reading it.